dino.icu.yaml

```
@@ -92,6 +92,11 @@ click: # a URL shortener
   type: CNAME
   value: cname.vercel-dns.com.
   
+codingcrew: # by https://github.com/shim-sham
+  ttl: 600 
+  type: CNAME
+  value: shim-sham.github.io/coding-crew. 
```

DNS records can't have paths in them.
```suggestion
  value: shim-sham.github.io.
```